cumulus/tasks/sf-sns-report/tests/index.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
```

Comment:
None of these tests are verifying that the correct message was sent to SNS.  They are only looking at the return value from the function.  Localstack has SNS support, please update the tests to verify that the correct messages are being sent to SNS.

Reply:
Updated the function to return the sns response.  If the MessageId is returned, then the message is sent to SNS.  There is no easy way to retrieve the message sent to SNS.